.github/maintainers/maintainers.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"text/template"
+
+	"github.com/bradleyfalzon/ghinstallation/v2"
+	"github.com/google/go-github/v69/github"
+)
+
+type TeamMember struct {
+	Login     string `json:"login"`
+	AvatarURL string `json:"avatar_url"`
+	HTMLURL   string `json:"html_url"`
+}
+
+func main() {
+	tr, err := ghinstallation.New(
+		http.DefaultTransport,
+		1185968, 63003777,
+		[]byte(os.Getenv("GITHUB_APP_PRIVATE_KEY")),
+	)
+	if err != nil {
+		log.Fatalf("Failed to create GitHub App transport: %v", err)
+	}
+
+	githubClient := github.NewClient(&http.Client{Transport: tr})
+
+	http.HandleFunc("GET /team/{slug}", func(w http.ResponseWriter, r *http.Request) {
+		teamSlug := r.PathValue("slug")
+		if teamSlug == "" {
+			http.Error(w, "Missing team slug", http.StatusBadRequest)
+			return
+		}
+
+		members, _, err := githubClient.Teams.ListTeamMembersBySlug(
```

Comment:
I don't know what order GitHub is using for this, but I spot-checked and it seems that the order returned here is the same order used for https://github.com/orgs/C2SP/teams/maintainers?query=membership%3Achild-team

I'm not a huge fan of that order as it has no visible reason (I'm guessing it's maybe ordered by internal GitHub ID), and would prefer alphabetical (either by name or username).

Reply:
Sorted alphabetically by username (since that's what we show at the moment).